tests/spki_tests.rs

```diff
@@ -0,0 +1,701 @@
+extern crate ring;
```

Comment:
Eventually we'll put lots of tests in tests/ and it would be nice to have some structure to the naming. Let's rename the file to `signature_spki_tests.rs`.

Also, please copy the license block to this file.